artifacts/experimental/etcd.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: etcd-pod
```

Comment:
rename this.  It's the core-etcd, right?

Reply:
Done